src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,236 @@
+---
+title: pub outdated
+description: Use pub outdated to help you update your package dependencies.
+diff2html: true
+---
+
+_Outdated_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+{% prettify nocode tag=pre+code %}
+$ pub outdated [options]
+{% endprettify %}
+
+Use `pub outdated` to identify out-of-date [package dependencies][]
+and get advice on how to update them.
+[Best practices for dependency management][best practices]
+include using the most recent stable package versions,
+so you can get the latest bug fixes and improvements.
+
+{{site.alert.version-note}}
+  The `pub outdated` command was introduced in Dart 2.8.
+{{site.alert.end}}
+
+## Overview
+
+Here's how you can use `pub outdated` to help you
+update the dependencies of a package that you own
+(whether it's an app or library package):
+
+1. If your package doesn't have a `pubspec.lock` file
+   checked into source control,
+   **run `pub get`** in the top directory of the package —
+   the directory that contains your package's
+  [`pubspec.yaml`](/tools/pub/pubspec) file.
+1. **Run `pub outdated`**
+   to identify which package dependencies are out-of-date.
+   Note the affected packages,
+   so that later you can test the behavior of code that uses them.
+1. Follow the recommendations of `pub outdated` for updating the packages.
+   Some updates might require only running `pub upgrade`.
+   Others might require updating `pubspec.yaml` before running `pub upgrade`.
+1. **Run `pub outdated`** to confirm that you've updated all the dependencies.
+1. **Test** your package to confirm that it still works as expected.
+
+
+## Example
+
+Here's an example of running `pub outdated` on
+[PENDING: link to source code]:
+
+```terminal
```

Comment:
I suggest you create an example that has one or more outdated main dependencies (that's the main issue wrt. shipping bugs etc.).

Reply:
I think we have resolved this?